api/src/opentrons/protocol_api/core/engine/labware.py

```diff
@@ -159,3 +160,12 @@ def get_well_core(self, well_name: str) -> WellCore:
             labware_id=self._labware_id,
             engine_client=self._engine_client,
         )
+
+    def get_deck_slot(self) -> Optional[DeckSlotName]:
+        """Get the deck slot the labware is in, if in a deck slot."""
```

Comment:
This docstring implies that you won't get the slot if the labware is on a module. Maybe this is better? 
```suggestion
        """Get the deck slot the labware is in, if on deck."""
```